src/donor_reporting_portal/config/settings.py

```diff
@@ -264,6 +264,13 @@
 }
 
 DEFAULT_FROM_EMAIL = 'donor_reporting_portal@unicef.org'
+EMAIL_BACKEND = 'unicef_notification.backends.EmailBackend'
+EMAIL_HOST = os.getenv('EMAIL_HOST', '')
```

Comment:
Any reason not to use the `env` object to get the ENV var, as used else where in the file?
e.g. `EMAIL_HOST = env('EMAIL_HOST', default='')`

Reply:
noted!